src/test/java/io/beanmapper/BeanMapperTest.java

```diff
@@ -348,7 +351,7 @@ void mapListCollectionWithNested() {
             add(new Address("MuisLaan", 1, "Frankrijk"));
         }};
         List<ResultAddress> targetItems =
-                beanMapper.map(sourceItems, ResultAddress.class);
+                (List<ResultAddress>) beanMapper.map(sourceItems, ResultAddress.class);
```

Comment:
This change, and similar changes would need to be reverted, due to reverting BeanMapper#map(Iterable, Class) back to BeanMapper#map(List, Class).